data/transaction/apiTransactionResult.go

```diff
@@ -52,6 +55,8 @@ type ApiTransactionResult struct {
 	Operation                         string                    `json:"operation,omitempty"`
 	Function                          string                    `json:"function,omitempty"`
 	IsRelayed                         bool                      `json:"isRelayed,omitempty"`
+	IsRefund                          bool                      `json:"isRefund,omitempty"`
+	InitiallyPaidFee                  string                    `json:"initiallyPaidFee,omitempty"`
```

Comment:
also here `InitiallyPaidFee` can be more near `Function` field

Reply:
Fixed.